contracts/pcv/utils/NamedStaticPCVDepositWrapper.sol

```diff
@@ -0,0 +1,180 @@
+pragma solidity ^0.8.4;
+
+import "../IPCVDepositBalances.sol";
+import "../../Constants.sol";
+import "../../refs/CoreRef.sol";
+import "@openzeppelin/contracts/utils/math/SafeCast.sol";
+
+/**
+  @notice a contract to report static PCV data to cover PCV not held with a reliable oracle or on-chain reading 
+  @author Fei Protocol
+
+  Returns PCV in USD terms
+*/
+contract NamedStaticPCVDepositWrapper is IPCVDepositBalances, CoreRef {
+    using SafeCast for *;
+
+    // -------------- Events ---------------
+    event BalanceUpdate(uint256 oldBalance, uint256 newBalance, uint256 oldFEIBalance, uint256 newFEIBalance);
+
+    event DepositRemoved(uint256 index);
```

Comment:
Should absolutely emit the address and probably depositName of the removed deposit. With indexing

Reply:
Also add a "DepositAdded" event

Reply:
added